tips/templates/tips/categories.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <script src="index.js"></script>
+    {% for tip in category_filtered_tips %}
+        <article class="post vt-post">
+            <div class="row">
+                <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4">
+                    <div class="post-type post-img">
+                        <a href="#"><img src="https://bootdey.com/img/Content/avatar/avatar1.png" class="img-responsive" alt="image post"></a>
```

Comment:
I think that the photos cause the page to look "heavy"
What do you think?

Reply:
I mark it as resolved since no one answred